tests/integration/test_case.go

```diff
@@ -97,6 +97,17 @@ type SchemaPatch struct {
 	ExpectedError string
 }
 
+type PatchCollection struct {
+	// NodeID may hold the ID (index) of a node to apply this patch to.
+	//
+	// If a value is not provided the patch will be applied to all nodes.
+	NodeID immutable.Option[int]
+
+	Patch string
```

Comment:
todo: document please

Reply:
My bad :)  Of course :)

- [ ] Doc test action Patch prop